docs/source/models/field_ionization_charge_state_prediction.py

```diff
@@ -0,0 +1,260 @@
+    'electric field' : sc.m_e**2 * sc.e**5 / (sc.hbar**4 * (4*sc.pi*sc.epsilon_0)**3),
+    'intensity' : sc.m_e**4 / (8 * sc.pi * sc.alpha * sc.hbar**9) * sc.e**12 / (4 * sc.pi * sc.epsilon_0)**6,
+    'energy' : sc.m_e*sc.e**4/(sc.hbar**2 * (4 * sc.pi * sc.epsilon_0)**2),
+    'time' : sc.hbar**3 * (4 * sc.pi * sc.epsilon_0)**2 / sc.m_e / sc.e**4
+}
+
+# (critical) barrier suppression field strength
+def F_crit(Z,E_Ip):
+    """
+    Classical barrier suppression field strength
+
+
+    :param Z: charge state of the resulting ion
+    :param E_Ip: ionization potential [unit: AU]
+
+    :returns: critical field strength [unit: AU]
+    """
+    return E_Ip**2. / (4. * Z)
+
+def n_eff(Z,E_Ip):
+    """
+    Effective principal quantum number
+
+    :param Z: charge state of the resulting ion
+    :param E_Ip: ionization potential [unit: AU]
+
+    :returns: effective principal quantum number
+    """
+    return Z / np.sqrt(2. * E_Ip)
+
+def ADKRate(Z,E_Ip,F):
+    """
+    Ammosov-Delone-Krainov ionization rate simplified by Stirling's approximation and m=0
+    like in publication [DeloneKrainov1998].
+
+    :param Z: charge state of the resulting ion
+    :param E_Ip: ionization potential [unit: AU]
+    :param F: field strength [unit: AU]
+    """
+    nEff = np.float64(n_eff(Z,E_Ip))
+    D = ((4. * Z**3.) / (F * nEff**4.))**nEff
+
+    rate = (np.sqrt((3. * nEff**3. * F) / (np.pi * Z**3.))
+          * (F * D**2.) / (8. * np.pi * Z)
+          * np.exp(-(2. * Z**3.)/(3. * nEff**3. * F)))
+
+    # set nan values due to near-zero field strengths to zero
+    rate = np.nan_to_num(rate)
+
+    return rate
+
+def convert_a0_to_Intensity(E_in_a0,lambda_laser=800.e-9):
+    """
+    :param E_in_a0: electric field [unit: a0]
+    :param lambda_laser: laser wavelength [unit: m]
+
+    :returns: intensity [unit: W/m²]
+    """
+
+    return (E_in_a0 / (0.85 * lambda_laser * 1e6))**2 * 1e22
+
+
+if __name__ == "__main__":
+    """
+    On execution this file produces the image
+    `field_ionization_charge_state_prediction.svg`
+    for the PIConGPU documentation.
+    """
+
+    # atomic units
+    AU_E_eV = atomic_unit['energy']/sc.electron_volt  # eV
+    AU_F = atomic_unit['electric field']  # V/m
+    AU_I = atomic_unit['intensity']  # W/m^2
+    AU_T = atomic_unit['time']  # s
+
+    # proton number: Neon
+    Z_max = 10
+    # array of possible charge states: Neon
+    Z = np.arange(Z_max) + np.int(1)
+    # array of ionization energies in eV for Neon
+    ionization_energies_eV  = np.array([
+    21.5645,
+    40.9630,
+    63.4233,
+    97.1900,
+    126.247,
+    157.934,
+    207.271,
+    239.0969,
+    1195.8078,
+    1362.1991
+    ])
+
+    # convert ionization energies to atomic units
+    i_pot_AU = ionization_energies_eV / AU_E_eV
+
+    """
+    Create the electric field distribution for our example.
+    """
+    #
+    lambda_laser = 800.e-9
+    # a0 in SI
+    a0 = np.float64(2*np.pi/lambda_laser * sc.m_e*sc.c**2 / sc.e)
```

Comment:
This is confusing - but I use it the same way 😨 
What you mean is the electric field strength of a_0 = 1.0.
Please add this as a comment above.

Reply:
I wasn't actually using this, anymore.
So I deleted it.